TACC/Core-Portal
# bug/FP-1003: Update Shared Workspace breadcrumb on title change

## Overview: ##
Update Shared Workspace breadcrumb on title change
## Related Jira tickets: ##

* [FP-1003](https://jira.tacc.utexas.edu/browse/FP-1003)

## Summary of Changes: ##
add a new action to setTtitleDescrition function on DataFilesPrjectList.js
## Testing Steps: ##
1. Go to My Dashboard
2. Go to My Data Files
3. Go to Shared Workspaces
4. Go to Any Workspace
5. Click Edit Description on Top Right
6. Change the workspace title and update changed
7. Check if breadcrumb path change instantly (DataFilesProjectFileListing module title change instantly so you can see if breadcrumb name is same as that)

## UI Photos:
![Screen Shot 2021-09-22 at 11 55 19 AM](https://user-images.githubusercontent.com/34716439/134387836-40c71275-8fbb-4cf4-9ddc-3cbce1ed5881.png)

## Notes: ##
